.github/workflows/checkdb.yml

```diff
@@ -1,11 +1,12 @@
 name: checkdb
 on:
   schedule:
-    - cron: '0 13 * * *'
+    - cron: '1 8 * * *'
```

Comment:
since pyquarkchain is already using 8:01 UTC, I suggest delaying ~5 hours so `0 13 * * *` is actually a good choice

Reply:
ok